lib/ecto/changeset.ex

```diff
@@ -429,7 +429,7 @@ defmodule Ecto.Changeset do
 
   def cast(_data, %{__struct__: _} = params, _permitted, _opts) do
     raise Ecto.CastError, type: :map, value: params,
-                          message: "expected params to be a :map, got: `#{inspect params}`"
+                          message: "expected params to be a :map, got: `#{inspect(params)}`"
```

Comment:
I guess Elixir formatter would wrap argument in parentheses, I noticed it and fixed it manually. Please tell if it's not desired.